docs/application/native/guides/app-management/app-controls.md

```diff
@@ -664,6 +669,126 @@ create_base_gui(appdata_s *ad)
    }
    ```
 
+<a name="action"></a>
+## Handle Application Control Actions
+
+You can define and handle each application control actions.
+
+- Defining the application control action
+
+    > **Note**
+    >
+    > From Tizen 5.5, the `id` attribute is added on the `<app-control>` element. You can define the name of the application control action using the `id` attribute of the `<app-control>` element.
+    >
+    > For example:
+    > ```
+    >  <ui-application appid="org.tizen.hello" exec="uiapp" hw-acceleration="on" launch_mode="single" multiple="false" nodisplay="false" taskmanage="true" type="capp">
+    >    <label>UI Application</label>
+    >    <icon>hello.png</icon>
+    >    <app-control id="dial">
+    >      <operation name="http://tizen.org/appcontrol/operation/dial"/>
+    >      <mime name="*"/>
+    >    </app-control>
+    >    <app-control id="dial-for-excel">
+    >      <operation name="http://tizen.org/appcontrol/operation/dial"/>
+    >      <mime name="application/vnd.ms-excel"/>
+    >    </app-control>
+    >  </ui-application>
+    >  ```
+- Handling each application control action
+
+  You can handle each application control action using `app_control_add_action_handler()`. The `app_control_action_cb()` callback function will be invoked if you register the action handler using the `app_control_add_action_handler()` function.
+
+   > **Note**
+   >
+   > If a matched action exists when an implicit launch request is received, the order of function call is:
+   > 1. `app_control_action_cb()`
+   > 2. `app_control_cb()`
+
+
+In this use case, the application registers the application control action handle based on the content, for example.
```

Comment:
Please use colon at the end of this line as it is following by a code block. Kindly rephrase as:
"...handle based on the content, for example:"